test/e2e-tests-standing-order.sh

```diff
@@ -21,7 +21,8 @@ truffle exec scripts/sell_order.js 1 2 1 1 1
 truffle exec scripts/wait_seconds.js 181
 retry -t 5 "mongo dfusion2 --eval \"db.accounts.findOne({'stateIndex': 6}).balances[1]\" | grep -2 2000000000000000000"
 
-# Cancel standing order
+# Update, then cancel standing order in same batch (make sure only cancel gets processed)
```

Comment:
Interesting example!

Reply:
I think the auction index need to be changed (0->2) to make the test effective

Reply:
Where is the auction index being referred to in this setting?

Reply:
I also don't understand. The values are: `(accountId, buyToken, sellToken, minBuy, maxSell)`

Reply:
sry